src/state/runner/reducers.js

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+// Current Simulation Runner
+import { createSlice } from '@reduxjs/toolkit';
+import { STATUSES } from '../../services/config/StatusConstants';
+import { RunnersUtils } from '../../utils';
+
+// Runners List
```

Comment:
Not sure this comment is still relevant, because the runner list and the "current runner" are all in the same state